### ISSUE TITLE
is there a way to find properties with split classification?

### ISSUE THREAD

**Issue by stevevance** [ccao-data/public#32]
earlier this year CCAO updated its policy on how it will assess certain split classification mixed-use buildings. I believe the classification is 3-18. 

my question is: what is the other classification that the property will have?
what data shows the assessment for each part of the property's improvements?

**Comment by ccao-jardine** [comment]
Howdy! The CCAO has actually been working on this update to these mixed use class 3-18s since 2023 (see the bottom of [this](https://www.cookcountyassessor.com/classifications-real-property) page). 

> what is the other classification that the property will have?

It depends; the basic idea is that if eligible to become a split class, then the residential portion will have a residential property class (typically, 2-36 or 3-97, depending on if the unit count exceeds 6) and the commercial portion will have a commercial property class (typically 5-17, but a 5-92 if more than one story, or other classes depending on the property). 

> what data shows the assessment for each part of the property's improvements?

Because these are income-producing properties, you can find data about their assessments on the Commercial Methodology Worksheets. As an example, for [North Chicago](https://www.cookcountyassessor.com/north-chicago-commercial-valuations), the Methodology Worksheet has a tab specifically to identify the split class properties. Each property has two rows: one for the residential portion, the other for the commercial portion. 

You can also see the property class codes for each portion on the [Assessor Property Detail](https://assessorpropertydetails.cookcountyil.gov/Datalets/Datalet.aspx?mode=commercial_bldg_cc&UseSearch=no&pin=14321040020000&jur=016&taxyr=2024&card=1&item=2&State=1|1|2|&items=2) page (the "big blue button" accessible from a PIN detail page). In the example I've linked, if you click on "Commercial Building" you can scroll through the "1 of 2 >" to see data for each portion. 

Does that help? 

FWIW, none of these are assessed by our residential AVM, so I moved it from that repo to this one.

**Comment by stevevance** [comment]
yes, this is helpful. I didn't know that the "Datalet" website showed the split classifications. 

I have a couple more questions:
1. In the North Chicago Methodology Worksheet, 14-32-104-002-0000 is shown as you said, one for residential portion and other for commercial portion. But the classes it shows are the same, 5-17. 
<img width="1140" alt="image" src="https://github.com/user-attachments/assets/6b4d127d-2a81-4b2b-9326-6ff4ee916207">


2. Will the split classifications show in the [Commercial Valuation Data](https://datacatalog.cookcountyil.gov/Property-Taxation/Assessor-Commercial-Valuation-Data/csik-bsws/explore/query/SELECT%0A%20%20%60keypin%60%2C%0A%20%20%60pins%60%2C%0A%20%20%60year%60%2C%0A%20%20%60township%60%2C%0A%20%20%60sheet%60%2C%0A%20%20%60class_es%60%2C%0A%20%20%60studiounits%60%2C%0A%20%20%60_1brunits%60%2C%0A%20%20%60_2brunits%60%2C%0A%20%20%60_3brunits%60%2C%0A%20%20%60_4brunits%60%2C%0A%20%20%60tot_units%60%2C%0A%20%20%60address%60%2C%0A%20%20%60adj_rent_sf%60%2C%0A%20%20%60aprx_comm_sf%60%2C%0A%20%20%60apt%60%2C%0A%20%20%60avgdailyrate%60%2C%0A%20%20%60bldgsf%60%2C%0A%20%20%60caprate%60%2C%0A%20%20%60carwash%60%2C%0A%20%20%60category%60%2C%0A%20%20%60ceilingheight%60%2C%0A%20%20%60cost_day_bed%60%2C%0A%20%20%60costapproach_sf%60%2C%0A%20%20%60covidadjvacancy%60%2C%0A%20%20%60ebitda%60%2C%0A%20%20%60egi%60%2C%0A%20%20%60excesslandarea%60%2C%0A%20%20%60excesslandval%60%2C%0A%20%20%60exp%60%2C%0A%20%20%60f_r%60%2C%0A%20%20%60finalmarketvalue%60%2C%0A%20%20%60finalmarketvalue_bed%60%2C%0A%20%20%60finalmarketvalue_key%60%2C%0A%20%20%60finalmarketvalue_sf%60%2C%0A%20%20%60finalmarketvalue_unit%60%2C%0A%20%20%60idphlicense%60%2C%0A%20%20%60incomemarketvalue%60%2C%0A%20%20%60incomemarketvalue_sf%60%2C%0A%20%20%60investmentrating%60%2C%0A%20%20%60land_bldg%60%2C%0A%20%20%60landsf%60%2C%0A%20%20%60nbhd%60%2C%0A%20%20%60netrentablesf%60%2C%0A%20%20%60noi%60%2C%0A%20%20%60oiltankvalue_atypicaloby%60%2C%0A%20%20%60owner%60%2C%0A%20%20%60parking%60%2C%0A%20%20%60parkingsf%60%2C%0A%20%20%60pctownerinterest%60%2C%0A%20%20%60pgi%60%2C%0A%20%20%60property_name_description%60%2C%0A%20%20%60property_type_use%60%2C%0A%20%20%60reportedoccupancy%60%2C%0A%20%20%60revenuebed_day%60%2C%0A%20%20%60revpar%60%2C%0A%20%20%60roomrev%60%2C%0A%20%20%60salecompmarketvalue_sf%60%2C%0A%20%20%60sap%60%2C%0A%20%20%60sapdeduction%60%2C%0A%20%20%60saptier%60%2C%0A%20%20%60stories%60%2C%0A%20%20%60taxdist%60%2C%0A%20%20%60taxpayer%60%2C%0A%20%20%60total2019revreported%60%2C%0A%20%20%60total2020revreported%60%2C%0A%20%20%60totalexp%60%2C%0A%20%20%60totalrev%60%2C%0A%20%20%60vacancy%60%2C%0A%20%20%60yearbuilt%60%2C%0A%20%20%60_2023permit_partial_demovalue%60%0AWHERE%20caseless_eq%28%60keypin%60%2C%20%2214-32-104-002-0000%22%29/page/filter) open data table?
Right now it's not and the same PIN shows it hasn't been updated since 2021. 
<img width="1505" alt="image" src="https://github.com/user-attachments/assets/b48365d5-6466-4d5d-8bfc-00171d36850a">

**Comment by ccao-jardine** [comment]
> In the North Chicago Methodology Worksheet, 14-32-104-002-0000 is shown as you said, one for residential portion and other for commercial portion. But the classes it shows are the same, 5-17.

Hm, my guess is that this excel sheet was produced before the class change was applied. On its [Property Detail Page](https://assessorpropertydetails.cookcountyil.gov/Datalets/Datalet.aspx?mode=commercial_bldg_cc&UseSearch=no&pin=14321040020000&jur=016&taxyr=2024&card=1&item=2&State=1|1|2|&items=2) it now shows split as 397 and 517.

> Will the split classifications show in the Commercial Valuation Data open data table?

Nice timing, we recently [updated](https://datacatalog.cookcountyil.gov/Property-Taxation/Assessor-Commercial-Valuation-Data/csik-bsws/about_data) it to include all Chicago 2024 commercial revaluation models. Split-class properties now have one row per class code.

**Comment by ccao-jardine** [comment]
Closing this out -- feel free to reopen if you think we haven't addressed your question. To the best of my knowledge, the above guidance is still the best way to programmatically identify split-class properties.
